docs/howto/how-to-debug.mdx

```diff
@@ -124,14 +125,34 @@ To resolve this issue:
 Verify that the relevant Traefik rule in the `docker-compose.yml` file is properly configured.
 
 2. Prune Docker Resources: kindly follow the commands below to stop and remove all the agenta containers; and to finally prune all the volumes associated with it. 
-- docker stop $(docker ps -q -f "name=agenta-*")
-- docker rm $(docker ps -aq -f "name=agenta-*")
-- docker volume ls -q -f "name=agenta-*"
-- docker volume rm VOLUME_NAME
 
-Once you have completed these steps, start the application again by running `docker-compose up` and access the backend server by making requests to `http://localhost:8000`. 
+```bash
+docker stop $(docker ps -q -f "name=agenta-*")
+docker rm $(docker ps -aq -f "name=agenta-*")
+docker volume ls -q -f "name=agenta-*"
+docker volume rm VOLUME_NAME
+```
+
+Once you have completed these steps, start the application again by running `docker compose up` and access the backend server by making requests to `http://localhost:8000`. 
 If the issue persists after following these troubleshooting steps, it's essential to investigate further into the Traefik configuration and the interaction between Traefik and the backend server.
 
+### Agenta Web Failing
```

Comment:
I would not recommend adding this to the documentation. The solution is valid only for this week (next week we'll not be using sqlite). The solution removes all the apps in the platform.

Reply:
True, I thought about that. 

How do we let our users know about how to resolve the issue? I've had two people run into the error since yesterday.

Reply:
Let's merge this into the docs then, and I will add a task in the refactoring issue to remove this troubleshooting docs to remove it

Reply:
Okay.